fairlearn/metrics/_metric_frame.py

```diff
@@ -189,6 +189,29 @@ class MetricFrame:
 
         .. deprecated:: 0.7.0
             `metric` will be removed in version 0.10.0, use `metrics` instead.
+
+    Examples
+    --------
+    >>> from fairlearn.metrics import MetricFrame, selection_rate
+    >>> y_true = [0,1,1,1,0,0,1,1]
+    >>> sex = ['Female']*4 + ['Male']*4
+    >>> selection_rates = MetricFrame(metrics=selection_rate,
+    ...                       y_true=y_true,
+    ...                       y_pred=y_true,
+    ...                       sensitive_features=sex)
+    >>> selection_rates.by_group
+    sensitive_feature_0
+    Female    0.75
+    Male       0.5
+    Name: selection_rate, dtype: object
+    >>> selection_rates.difference()
+    0.25
+    >>> selection_rates.group_max()
+    0.75
+    >>> selection_rates.group_min()
+    0.5
+    >>> selection_rates.ratio()
+    0.6666666666666666
```

Comment:
If you want to avoid the recurring decimal (and if you look in the other docs, something like `0.666...` should be a neater alternative in case Python ever changes its default output format), then you could extend the array to 10 items, which should allow the ratio to be a finite decimal.

Reply:
Should be fixed now!